WcaOnRails/app/models/competition.rb

```diff
@@ -222,6 +235,7 @@ def update_foreign_keys_when_id_changes
       CompetitionMedium.where(competitionId: id_was).update_all(competitionId: id)
       CompetitionDelegate.where(competition_id: id_was).update_all(competition_id: id)
       CompetitionOrganizer.where(competition_id: id_was).update_all(competition_id: id)
+      DelegateReport.where(competition_id: id_was).update_all(competition_id: id)
```

Comment:
Cool! I did not know about that `reflect_on_all_associations`. It's also not super clear to me what the right way is to get a list of all models is.

I'm going to declare this "not worth the bother", as we should even *have* mutable foreign keys. See #381.